src/screens/CardInfo/CardInfoScreen.tsx

```diff
@@ -0,0 +1,216 @@
+    <View style={StyleBase.container}>
+      <StatusBar barStyle='default' />
+      <TopBar iconLeft={'arrow-left'} onPressLeft={() => onPressBack(navigation)} />
+      {/* todo: apply paddings
+                image as Card.businessDetails.iconImageId */}
+      {/*temp solution - Tile of CardTile style override */}
+      <View style={styles.cardTile}>
+        <CardTile
+          image={require('../../assets/images/biedronka_homepage.jpg')}
+          onPress={() => {}}
+        />
+      </View>
+      <Tile style={styles.cardInfo}>
+        <View style={styles.accountTileContainer}>
+          <Text style={styles.text}>Account balance</Text>
+          <Text style={styles.text}>100</Text>
+        </View>
+      </Tile>
+      <View style={styles.buttonsContainer}>
+        {/* todo: custom button styles to fit screen
+                          or make them cardtiles instead but without image?
+                          implement onPress   */}
+        <Pressable
+          onPress={() => {
+            setState(1);
+          }}
+        >
+          <Tile style={styles.button}>
+            <Text style={styles.text}>Business</Text>
+          </Tile>
+        </Pressable>
+        <Pressable
+          onPress={() => {
+            setState(0);
+          }}
+        >
+          <Tile style={styles.button}>
+            <Text style={styles.text}>Benefits</Text>
+          </Tile>
+        </Pressable>
+        {/*
+        <CustomButton onPress={() => alert('Work in progress')} title='Business' type='primary' />
+                        <CustomButton onPress={() => alert('Work in progress')} title='Benefits' /> */}
+      </View>
+      {/* everything below is in temp form, what is rendered will be based on button^ pressed */}
+      {screenState ? (
+        <View style={styles.container}>
+          <BoxContainer style={styles.boxContainer}>
+            <Text style={[styles.text, { paddingBottom: 40 }]}>Business name</Text>
+            <Text style={[styles.text, { paddingBottom: 40 }]}>Address</Text>
+            <Text style={styles.text}>Info</Text>
+          </BoxContainer>
+          {/* todo: apply proper paddings
+                even more temp solutions below */}
+          <View style={styles.claimButton}>
+            <CustomButton onPress={() => alert('Work in progress')} title='Claim Benefits' />
+          </View>
+          <View style={styles.showCard}>
+            <CustomButton onPress={() => alert('Work in progress')} title='Show Card' />
+          </View>
+        </View>
+      ) : (
+        <View style={styles.container}>
+          <View style={styles.benefitsContainer}>
+            <Text style={styles.text}>List of benefits</Text>
+            {/*todo: style the tiles in list */}
+            <BenefitList
+              benefits={benefits}
+              onPress={() => {
+                //to add: passing Benefit object
+                navigation.push('BenefitDescriptionScreen');
```

Comment:
wolałbym, żeby to byly zaimportowane constanty z `routes`.
`BENEFIT_DESCRIPTION_ROUTE` - jak nam się odmieni i nagle będziemy chcieli innego patha to nie trzeba będzie szukać tego stringa po całej aplikacji.